src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -805,6 +814,313 @@ TEST_F(MessageHelperTest,
+  policy_test::MockPolicyHandlerInterface policy_handler;
+
+  EXPECT_CALL(policy_handler, PolicyEnabled()).WillOnce(Return(true));
+  EXPECT_CALL(policy_handler, CheckSystemAction(_, _)).WillOnce(Return(true));
+  EXPECT_CALL(*app_shared_ptr, policy_app_id()).WillOnce(Return(""));
+  EXPECT_EQ(MessageHelper::ProcessSoftButtons(message_params,
+                                              app_shared_ptr,
+                                              policy_handler,
+                                              mock_application_manager),
+            mobile_apis::Result::SUCCESS);
+}
+
+TEST_F(MessageHelperTest, CreateAddVRCommandToHMI_ValidVrCommand_SUCCESS) {
+  const uint32_t kCmdId = 1;
+  const smart_objects::SmartObject kVrCommands;
+  const uint32_t kAppId = 1;
+  const uint32_t kGrammarId = 2;
+  application_manager_test::MockApplicationManager mock_application_manager;
+  MockApplicationSharedPtr app_shared_ptr =
+      utils::MakeShared<MockApplication>();
+  smart_objects::SmartObjectSPtr valid_vr_command =
+      utils::MakeShared<smart_objects::SmartObject>(
+          smart_objects::SmartType_Map);
+  (*valid_vr_command)[strings::params][strings::function_id] =
+      hmi_apis::FunctionID::VR_AddCommand;
+  smart_objects::SmartObject msg_params =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+  msg_params[strings::cmd_id] = kCmdId;
+  msg_params[strings::app_id] = kAppId;
+  msg_params[strings::grammar_id] = kGrammarId;
+  msg_params[strings::type] = hmi_apis::Common_VRCommandType::Command;
+  (*valid_vr_command)[strings::msg_params] = msg_params;
+
+  EXPECT_CALL(mock_application_manager, GetNextHMICorrelationID())
+      .WillOnce(Return(kAppId));
+  EXPECT_CALL(mock_application_manager, application(kAppId))
+      .WillOnce(Return(app_shared_ptr));
+  EXPECT_CALL(*app_shared_ptr, get_grammar_id()).WillOnce(Return(kGrammarId));
+
+  smart_objects::SmartObjectSPtr vr_command =
+      MessageHelper::CreateAddVRCommandToHMI(
+          kCmdId, kVrCommands, kAppId, mock_application_manager);
+
+  EXPECT_EQ((*valid_vr_command)[strings::msg_params][strings::cmd_id],
+            (*vr_command)[strings::msg_params][strings::cmd_id]);
+  EXPECT_EQ((*valid_vr_command)[strings::msg_params][strings::app_id],
+            (*vr_command)[strings::msg_params][strings::app_id]);
+  EXPECT_EQ((*valid_vr_command)[strings::msg_params][strings::grammar_id],
+            (*vr_command)[strings::msg_params][strings::grammar_id]);
+  EXPECT_EQ((*valid_vr_command)[strings::msg_params][strings::type],
+            (*vr_command)[strings::msg_params][strings::type]);
+  EXPECT_EQ((*valid_vr_command)[strings::params][strings::function_id],
+            (*vr_command)[strings::params][strings::function_id]);
+}
+
+TEST_F(MessageHelperTest, CreateAddVRCommandToHMI_NullAppId_SUCCESS) {
+  const uint32_t kCmdId = 0;
+  const smart_objects::SmartObject kVrCommands;
+  const uint32_t kAppId = 0;
+  const uint32_t kGrammarId = 2;
```

Comment:
@okozlovlux Adde prefix `u` to all unsigned variables. You did it somewhere, somewhere - not. Keep consistency.